project/CONTRIBUTING.md

```diff
@@ -87,9 +87,16 @@ Some rules have to be respected about the test:
   * `@codeCoverageIgnore`
   * `@codeCoverageIgnoreStart`
   * `@codeCoverageIgnoreEnd`
+  * `@expectedException`
+  * `@expectedExceptionMessages`
 * All test methods should be prefixed by `test`. Example: `public function testItReturnsNull()`.
+* All test method names must be in camel case format
 * As opposed, the `@test` annotation is prohibited.
 * Most of the time, the test class should have the same name as the targeted class, suffixed by `Test`.
+* `@expectedException*` annotations are prohibited in favor of:
+  * the `setExpectedException` method if phpunit 4 is still in use;
```

Comment:
* `the` -> `The`
* `setExpectedException` -> `TestCase::setExpectedException` (I think it's `TestCase`, please double check. :+1:)

Same for line below.